src/lib/customers/20minuten-de.json

```diff
@@ -0,0 +1,54 @@
+{
+	"title": "bespinian unterstützt 20 Minuten bei der Modernisierung der digitalen Produkte",
+	"logo": {
+		"file": "tx-group.webp",
+		"alt": "TX Group"
+	},
+	"customer": "<a href=\"https://www.20min.ch/\" target=\"_blank\">20 Minuten</a> ist die reichweitenstärkste Medienmarke der Schweiz. Sie gehört wie auch das Medienhaus Tamedia und die Werbevermarkterin Goldbach zur TX Group. 20 Minuten zeichnet sich durch ihre Reichweite in allen Sprachregionen und dem Storymix aus News, Unterhaltung und Inspiration aus und war/ist für bespinian ein wichtiger Partner in der Zusammenarbeit bezüglich der Migration der digitalen Produkte auf eine neue Infrastruktur, und anderen Projekten im Rahmen von OAuth2, Cloudifizierung, Containerisierung und Modernisierung von Software.",
+	"background": "20 Minuten ist die schweizweit grösste Pendlerzeitung und eine der meistbesuchten Webseiten der Schweiz. Es ist ein Medienportal mit täglichen Beiträgen aus dem Bereich News, Unterhaltung und Inspiration. 20 Minuten hatte 2019 eine grosse Migration ihrer Applikationen weg von der Legacy Infrastruktur und dem teilweise fast 20-jährigen Applikationscode geplant. Dabei ging es um eine Migration von bestehenden VMs auf eine moderne Kubernetes-basierte Containerinfrastruktur auf AWS. Das Backend sollte von Perl auf Go migriert und containerisiert werden. Das Frontend wurde ebenfalls von Perl gerendert und sollte in eine moderne Next.js Applikation umgewandelt werden. 20 Minuten hat täglich über eine Million Nutzer:innen. Dies macht eine Migration ohne Downtime besonders anspruchsvoll. Zusätzlich, war es wichtig, die Betriebskosten trotz dieses grossen Loads in einem überschaubaren Rahmen zu halten.",
```

Comment:
Replace `in einem überschaubaren Rahmen` by `in einem verträglichlichen Rahmen` or `in einem akzeptablen Rahmen`